src/_components/GetInvolvedCard.jsx

```diff
@@ -7,7 +7,7 @@ export default function GetInvolvedCard({
   redirect_url,
 }) {
   return (
-    <figure className="flex flex-col w-full border border-black rounded-lg lg:p-5 p-8 lg:max-w-[445px] h-[450px]">
+    <figure className="flex flex-col border border-black rounded-lg lg:p-5 p-5 lg:h-[450px]">
```

Comment:
I think adding `md:w-1/3` will keep the cards consistent. Between the small and large breakpoint it is ok if they don't remain the same size

Reply:
Can you also change the title text size to `text-3xl` and the description text size to `text-xl`. The `lg` breakpoint text can be left the same. I was playing around and this helped avoid content dropping and being uneven